datachannel.go

```diff
@@ -125,15 +125,13 @@ func (c *DataChannel) BufferedAmount() int {
 	return int(C.CGO_Channel_BufferedAmount(c.cgoChannel))
 }
 
-// TODO: Variadic options constructor, probably makes more sense for
-// CreateDataChannel in parent package PeerConnection.
-type Init struct {
```

Comment:
A breaking change here is probably ok, since this is currently ignored in `peerconnection.cc`

> `DataChannelInit *r = (DataChannelInit*)dict;`